components/card/card.css

```diff
@@ -0,0 +1,106 @@
+@import url("/docs/main.css");
+
+.card {
+    min-height: fit-content;
+    width: 60%;
+    justify-content: flex-start;
+    box-shadow: 2px 2px 10px grey;
```

Comment:
you can make a variable of shadow s

Reply:
Oh I forget that. Thanks. for reminding!